examples/deploy-k8s/README.md

```diff
@@ -0,0 +1,60 @@
+# Deploy Ollama to Kubernetes
```

Comment:
How's `examples/kubernetes` for the directory name? This can be the initial example and future ones can use `examples/kubernetes-<xyz>`